challenges/readmes/tree.md

```diff
@@ -83,6 +83,11 @@ This takes any value as an argument and adds it to the tree in to appropriate po
 
 Return a boolean letting you know if a specified value is within the tree.
```

Comment:
Looks like you did your work on main, don't forget to push your work onto a branch!